.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "editor.formatOnSave": true,
+
+  "[javascript]": {
+    "editor.formatOnSave": false
+  },
+
+  "editor.codeActionsOnSave": {
```

Comment:
We should sort this list as follows:

1. editor.codeActionsOnSave
2. editor.formatOnSave
3. prettier.disableLanguages
4. [javascript]